### PR TITLE
Gui 2805 allow access to csv usage reports by service.

### PR DESCRIPTION
**Pull request by dkavanagh**
https://eucalyptus.atlassian.net/browse/GUI-2805

### CODE REVIEW

**Comment by aliceh** [comment]
Retest his please

**Comment by aliceh** [comment]
Retest this please

**Comment by aliceh** [comment]
The cloud against which we are running the tests is broken. I am in the process of replacing the cloud/debugging. I will restart test when that is done. Sorry for the inconvenience.

**Comment by aliceh** [comment]
Retest this please

File: eucaconsole/static/js/pages/usage_reports.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+                    }
+                };
+                vm.isDownloading = false;
+                vm.downloadCSV = function ($event) {
+                    $event.preventDefault();
+                    if ($scope.downloadReportForm.$invalid) {
+                        return false;
+                    }
+                    vm.isDownloading = true;
+                    params = {
```
Comment (kamalgill) [inline eucaconsole/static/js/pages/usage_reports.js]: Missing a var here

**Pull request merged**